apps/wing/src/cli.ts

```diff
@@ -152,6 +152,7 @@ async function main() {
         .default("sim")
     )
     .option("-p, --plugins [plugin...]", "Compiler plugins")
+    .option("--no-clean", "Disable resource cleaning")
```

Comment:
I'm not sure the word "resource" is clear here. Maybe something like this:

```suggestion
    .option("--no-clean", "Keep build output")
```